assembl/static2/js/app/components/debate/thread/answerForm.jsx

```diff
@@ -43,9 +47,9 @@ class AnswerForm extends React.PureComponent<*, AnswerFormProps, AnswerFormState
       submitting: false
     };
   }
-
```

Comment:
From previous discussion with @cedricmessiant @vincentfretin, the team consensus was to have spaces between methods in JS. We should make this a rule as part of the linter so that it catches it.

Reply:
Same here. I'm not sure to understand :(